spec/tree_spec.rb

```diff
@@ -3,12 +3,41 @@
 
 describe Tree do
   it 'should be a Class' do
-    expect(described_class.is_a? Class).to eq true
+    expect(described_class.is_a?(Class)).to eq true
+  end
+  it 'should be dead at the age of 11' do
+    10.times { subject.age! }
+    expect(subject).to_not be_dead
+    subject.age!
+    expect(subject).to be_dead
+  end
+  it 'should age if asked' do
+    prev_value = subject.age
+    subject.age!
+    expect(subject.age).to be > prev_value
+  end
+  it 'should not have any apples at birth' do
+    expect(subject.any_apples?).to eq false
+  end
+  it 'should raise an error when picking an empty tree' do
+    expect { subject.pick_an_apple! }.to raise_error(NoApplesError)
+  end
+  it 'should have apples after adding' do
+    subject.add_apples
+    expect(subject.any_apples?).to eq true
```

Comment:
Is there an implicit definitition of the subject going on here?  Can't hurt to make it explicit, unless I'm missing it somehow.